packages/ckeditor5-utils/src/translation-service.ts

```diff
@@ -196,8 +199,9 @@ export function _translate( language: string, message: Message, quantity: number
 		return message.string;
 	}
 
-	const dictionary = global.window.CKEDITOR_TRANSLATIONS[ language ].dictionary;
-	const getPluralForm = global.window.CKEDITOR_TRANSLATIONS[ language ].getPluralForm || ( n => n === 1 ? 0 : 1 );
+	const dictionary = ( translations ? translations : global.window.CKEDITOR_TRANSLATIONS )[ language ].dictionary;
+	const getPluralForm = ( translations ? translations : global.window.CKEDITOR_TRANSLATIONS )[ language ].getPluralForm ||
```

Comment:
This ternary conditional check is repeated twice:

```
translations ? translations : global.window.CKEDITOR\_TRANSLATIONS
```

Lets let's make it as a constant and use it.